CI.hs

```diff
@@ -30,13 +31,15 @@ main = do
             <> Opts.progDesc "Build (and possibly upload) ghc-lib and ghc-lib-parser tarballs."
             <> Opts.header "CI - CI script for ghc-lib"
           )
-    Options { upload, ghcFlavor, stackOptions } <- Opts.execParser opts
-    version <- buildDists ghcFlavor stackOptions
-    when upload $ bintrayUpload version
+    Options { ghcFlavor, daMergeBaseSha, daPatches, daFlavor, stackOptions } <- Opts.execParser opts
```

Comment:
At the least I'd like to see the DA options bundled into a structure ala `stackOptions`. I think that structure should be an optional only defined in the presence of a DA build rather than always defined with defaults.

Reply:
I agree but let’s please do that separately.